src/shared/api/helpers.ts

```diff
@@ -14,6 +15,28 @@ export interface NetworkErrorObject {
   error?: Error
 }
 
+export function rejectNetworkError(error: NetworkErrorObject) {
+  // only capture network errors if they are not a rate limit error
+  // this will typically only be schema parsing errors
+  if (error.status !== 429 && error.dev && error.error) {
+    Sentry.withScope((scope) => {
+      scope.addBreadcrumb({
+        category: 'network.error',
+        level: 'error',
+        data: error.error,
```

Comment:
the previous [version](https://github.com/codecov/gazebo/blob/82b3d3ef75c7ff01919953fa41dbeb5b7836d104/src/layouts/shared/NetworkErrorBoundary/NetworkErrorBoundary.jsx#L183) had also included `message: error.dev` in the "breadcrumb" if we wanted to keep that in (honestly not sure where that shows up in Sentry dashboard, but based on [this](https://docs.sentry.io/product/issues/issue-details/breadcrumbs/#breadcrumb-attributes) it seems like it can with search?)

Reply:
Nice catch, I'll add that back in